### PR TITLE
building evc pipeline

### PR DESCRIPTION
**Pull request by DivineTaminangHO**
# WHAT ?
Pipeline for EVC
E-Visa Contact Us is a form for members of the public to contact the home office regarding their e-visa

# WHY ?
This pipeline will allow for the applications to be developed in or to enable applicant to contact the home office 

# HOW ?
A new evc pipeline files including drone.yaml has been created. Also a hof-services-config file for this service has also been created. These will enable the pipeline steps to run on the drone UI. 
And after it is merged to master, a developers will create their branch from master and add their codes.

### CODE REVIEW

**Comment by adityababumallisettiHO** [comment]
Can you also add sonarqube properties file please

**Review by adityababumallisettiHO — CHANGES_REQUESTED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+type: kubernetes
+
+environment:
+  APP_NAME: evc
+  PROD_ENV: sas-evc-prod
+  STG_ENV: sas-evc-stg
+  BRANCH_ENV: sas-evc-branch
+  UAT_ENV: sas-evc-uat
+  #PRODUCTION_URL:
+  #IMAGE_URL:
```
[the remark this answers is not in the document]
Comment (DivineTaminangHO) [reply .drone.yml:13]: done

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+    image: plugins/slack
+    settings:
+      channel: sas-hof-security
+      failure: ignore
+      icon_url: http://readme.drone.io/0.5/logo_dark.svg
+      icon.url: http://readme.drone.io/0.5/logo_dark.svg
+      template: "CRON Job {{build.deployTo}} of E-Visa Contact Us has {{build.status}} - <{{build.link}}|#{{build.number}}> {{#success build.status}}\n  :thumbsup: :thumbsup: :thumbsup:\n{{else}}\n  :x: :x: :x:\n{{/success}} Author: {{build.author}}\n\nDuration: {{since job.started}}\n\nJob: <{{build.link}}|#{{build.number}}>\n\nCommit: {{build.commit}}\n"
+      username: Drone
+      webhook:
+        from_secret: slack_webhook
```
[the remark this answers is not in the document]
Comment (DivineTaminangHO) [reply .drone.yml:349]: yes, these are from coa and I already have the slack_webhook configured In drone UI

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+
+  - name: image_to_ecr
+    image: plugins/ecr
+    settings:
+      access_key:
+        from_secret: aws_access_key_id
+      secret_key:
+        from_secret: aws_secret_access_key
+      region: eu-west-2
+      #repo: ${IMAGE_REPO}
```
[the remark this answers is not in the document]
Comment (DivineTaminangHO) [reply .drone.yml]: resolved

File: .drone.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (DivineTaminangHO) [reply .drone.yml]: resolved

File: .drone.yml [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+    image: plugins/slack
+    settings:
+      channel: sas-build
+      failure: ignore
+      icon_url: http://readme.drone.io/0.5/logo_dark.svg
+      icon.url: http://readme.drone.io/0.5/logo_dark.svg
+      template: "CRON Job {{build.deployTo}} of E-Visa Contact Us has {{build.status}} - <{{build.link}}|#{{build.number}}> {{#success build.status}}\n  :thumbsup: :thumbsup: :thumbsup:\n{{else}}\n  :x: :x: :x:\n{{/success}} Author: {{build.author}}\n\nDuration: {{since job.started}}\n\nJob: <{{build.link}}|#{{build.number}}>\n\nCommit: {{build.commit}}\n"
+      username: Drone
+      webhook:
+        from_secret: slack_webhook
```
[the remark this answers is not in the document]
Comment (DivineTaminangHO) [reply .drone.yml]: resolved

**Review by adityababumallisettiHO — APPROVED** [review]
lgtm

**Pull request merged**